policy/modules/system/getty.te

```diff
@@ -48,6 +48,7 @@ logging_log_filetrans(getty_t, getty_log_t, file)
 
 manage_files_pattern(getty_t, getty_runtime_t, getty_runtime_t)
 files_pid_filetrans(getty_t, getty_runtime_t, file)
+getty_watch_runtime_dir(getty_t)
```

Comment:
Since the interface is used here, it doesn't seem to warrant an interface yet.

Reply:
ACK